db/migrate/20230818104713_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :posts_counter
+      t.timestamps
```

Comment:
Great job @haftamudesta 🤝

- [ ] As stated in the project requirements, kindly ensure the `role` column exists in the users' table.

Reply:
@1GORDON, the role column is already there in the user table. kindly double check it.
![image](https://github.com/haftamudesta/Blog-App/assets/99684109/84755df8-056c-43ab-bfd4-169845e3b61d)